### PR TITLE
[RPA-1302] Zip: unzip file

### PR DESCRIPTION
**Pull request by seul88**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

File: runbotics/runbotics-orchestrator-ui/src/src-app/Actions/zip.actions.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                'ui:order': ['input']
+            },
+            formData: {
+                input: {
+                    path: undefined
+                }
+            }
+        }
+    }
+});
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/Actions/zip.actions.ts]: no tooltips?

**Review by star-in-the-jar — APPROVED** [review]

**Comment by star-in-the-jar** [comment]
bump version

File: runbotics/runbotics-desktop/src/action/zip/zip.action-handler.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    ) {
+        super();
+    }
+
+    async unzipFile(input: UnzipFileActionInput) {
+        const { path, fileName } = input;
+
+        try {
+            const fullPath = `${path}${pathPackage.sep}${fileName}`;
+            const zip = new AdmZip(`${fullPath}.zip`);
```
Comment (preludium) [inline runbotics/runbotics-desktop/src/action/zip/zip.action-handler.ts]: if `path` is not provided you get `undefined` in fullPath

File: runbotics/runbotics-desktop/src/action/zip/zip.types.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { DesktopRunRequest } from '@runbotics/runbotics-sdk';
+import { ZipAction } from 'runbotics-common';
+
+export type UnzipFileActionInput = {
+    path: string;
```
Comment (preludium) [inline runbotics/runbotics-desktop/src/action/zip/zip.types.ts]: should be optional

**Review by preludium — CHANGES_REQUESTED** [review]

**Review by preludium — APPROVED** [review]

**Pull request merged**